### PR TITLE
[CLEANUP] Autoformat the code

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Review by JakeQZ — CHANGES_REQUESTED** [review]
Just one alignment issue; otherwise fine.

**Comment by oliverklee** [comment]
> Just one alignment issue

Which one?

File: tests/CSSList/AtRuleBlockListTest.php [hunk 1]
```diff
@@ -35,17 +35,19 @@ public static function provideSyntacticallyCorrectAtRule(): array
...
+                        .flex-container {
+                            display: flex;
+                        }
                     }
-                    .flex-container {
-                        display: flex;
-                    }
-                }
-            '],
+            ',
```
Comment (JakeQZ) [inline tests/CSSList/AtRuleBlockListTest.php:49]: The closing quote should be indented to align with the opening quote.
```suggestion
                ',
```

**Review by JakeQZ — CHANGES_REQUESTED** [review]
Sorry, failed to submit the comment.

**Review by JakeQZ — APPROVED** [review]

**Pull request merged**